### ISSUE TITLE
intermittent test failure (TimeoutSuite)

### ISSUE THREAD

**Issue by SethTisue** [scalameta/munit#31]
not sure how often it happens, but I did see it once already: https://scala-ci.typesafe.com/job/scala-2.12.x-jdk14-integrate-community-build/699/artifact/logs/munit-build.log

```
[munit] [91m==> X [0m[91mmunit.TimeoutSuite[0m.[91mfast[0m  [90m0.025s[0m java.util.concurrent.TimeoutException: Futures timed out after [10 milliseconds]
```

**Comment by olafurpg** [comment]
Thank you for reporting! We can increase the timeout for that test. We have already run this test 84 times in our GitHub Actions CI without any flaky failures

<img width="848" alt="Screenshot 2020-01-29 at 13 17 47" src="https://user-images.githubusercontent.com/1408093/73359958-dccbe180-4299-11ea-84fd-9ad35cc7e7be.png">

https://scalameta.org/munit/docs/reports.html

**Comment by olafurpg** [comment]
I opened https://github.com/scalameta/munit/pull/32 increasing the timeout from 10ms to 100ms

**Comment by olafurpg** [comment]
Please let me know if the problem persists. I'm not sure if there is a better way to test functionality like this, async code is generally quite tricky to test

**Comment by SethTisue** [comment]
scala/community-build@2b61ffb6a47fdfd89639d8edd44183e736a776d0 pulls in the fix (and I also merged it forward onto 2.12.x and 2.13.x)

if I ever see the test fail again, I'll report it here
